src/components/BackButton.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div class="back-button" :style="`left:${x}; top:${y};`" v-on:click="nav">
+        <span id="arrow">{{arrow}}</span>
+    </div>
+</template>
+
+<script>
+export default {
+    props: ["x", "y","direction"],
+    data:function(){
+        return{arrow:(this.direction=="forward"?'→':'←')}
+    },
+    methods:{
+        nav(){
+            this.$router.go(this.direction=="forward"? 1 : -1)
+        }
+    },
+};
```

Comment:
Stay consistent with spacing:

- For props, there's a space after the colon. For data and methods, there is not.
- There's spaces between parts of your ternary in some places but not all.